# This message contains a list of motion planning constraints.

motion_planning_msgs/JointConstraint[] joint_constraints
motion_planning_msgs/PositionConstraint[] position_constraints
motion_planning_msgs/OrientationConstraint[] orientation_constraints
motion_planning_msgs/VisibilityConstraint[] visibility_constraints
